demo/infra/src/application/corpus/pipeline/handlers/inventory.ts

```diff
@@ -128,11 +128,9 @@ export function convertBucketInventoryToManifest(
   inventory: BucketInventory,
   include?: Set<string>,
 ): BucketInventoryManifestFile {
-  const prefix = (inventory.prefix ?? '').replace(/^\\/, '');
-
   const contents = include == null ? inventory.contents : inventory.contents.filter((v) => include.has(v.Key));
 
-  return [{ prefix: `s3://${inventory.bucket}/${prefix}` }, ...contents.map((v) => v.Key)];
+  return [{ prefix: `s3://${inventory.bucket}/` }, ...contents.map((v) => v.Key)];
```

Comment:
[flagged]: why?

Reply:
The reason I removed the prefix from the manifest file is because the bucket prefix is actually part of the files. If we add bucket prefix to prefix field, the actual file path will become `s3://${bucket}//${prefix}/${prefix}/file`. The SageMaker processing job will throw 404 error complaining not able to find the files. I'll add a comment in the code.